htdocs/fourn/commande/dispatch.php

```diff
@@ -258,7 +258,7 @@
 			}
 
 			// We ask to move a qty
-			if (GETPOST($qty) != 0) {
+			if (GETPOSTINT($qty) != 0) {
```

Comment:
GETPOST means the entry come from a parameter so from a user input. GETPOSTINT will not be enough
We must converte a text input into a numeric value.
So good fix should be
`if (price2num(GETPOST($qty), 'MS') != 0) {`

Can you try with this ?